daffodil-core/src/main/scala/org/apache/daffodil/dsom/IIBase.scala

```diff
@@ -180,7 +180,17 @@ abstract class IIBase( final override val xml: Node, xsdArg: XMLSchemaDocument,
     res
   }.value
 
-  final lazy val schemaLocationProperty = getAttributeOption("schemaLocation")
+  final lazy val schemaLocationProperty = {
+    val prop = getAttributeOption("schemaLocation")
+    prop.map { text =>
+      if (text.contains("edu/illinois/ncsa/daffodil")) {
+        SDW("schemaLocation property uses deprecated edu/illinois/ncsa/daffodil path instead of org/apache/daffodil. Converting to new path.")
+        text.replace("edu/illinois/ncsa/daffodil", "org/apache/daffodil")
+      } else {
+        text
+      }
+    }
+  }
```

Comment:
Actual code change other than package rename. Provides backwards compatibility for schemas using old imports.